### DISCUSSION TITLE
How to configure the Docker container that Porter runs to use host networking?

### DISCUSSION THREAD

**Discussion by cgiraldo** [getporter/porter#1366] [Q&A] [answered]
In https://porter.sh/best-practices/kind/ there are two options:

1. Configure the Docker container that Porter runs to use host networking
2. Configure the KinD cluster to use an IP address that is resolvable from within Porter’s Docker container

And option 2 is explained but, how to configure option 1?
I can see `--allow-docker-host-access`  flag in porter install, but this does not configure host networking.

**Comment by vdice** [comment, marked as answer]
Hi @cgiraldo, Oops, indeed, as you've pointed out, option 1 isn't currently supported by Porter.  That section of the kind doc is therefore misleading.  I'll submit a PR to update here shortly.

Meanwhile, it looks like we have a ticket around this area of functionality in https://github.com/getporter/porter/issues/1347.  It'd be great if you have a minute to check it out and perhaps add your use case(s) to it as well.

**Reply by cgiraldo** [reply]
Hi @vdice and thank you for your great response. 

I am just trying things in a microk8s, however porter.sh does not work with microk8s.kubeconfig (kubernetes api host is set to 127.0.0.1). 
I have updated my kubeconfig and microk8s certs with a hostname instead of 127.0.0.1, but the hostname must be resolvable in the docker container that Porter runs. updating /etc/hosts make my local kubectl work, but not inside porter container.

I will update the issue with my use case.
